verifiersdemo/source/objc/EndpointOwnershipDemo/KaaManager.m

```diff
@@ -0,0 +1,139 @@
+/**
+ *  Copyright 2014-2016 CyberVision, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#import "KaaManager.h"
+
+@interface KaaManager () <ProfileContainer, ConfigurationDelegate, KaaClientStateDelegate>
+
+@property (nonatomic, strong) volatile id<KaaClient> kaaClient;
+@property (nonatomic, strong) KAAKaaVerifiersTokens *verifiersTokens;
+
+@end
+
+@implementation KaaManager
+
+/**
+ * Returns shared instance of KaaManager class.
+ */
+
++ (KaaManager *)sharedInstance {
+    static KaaManager *manager = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        manager = [[KaaManager alloc] init];
+        manager.kaaClient = [KaaClientFactory clientWithContext:[[DefaultKaaPlatformContext alloc] init] stateDelegate:manager];
+        [manager.kaaClient setProfileContainer:manager];
+    });
+    return manager;
+}
+
+- (void)startKaaClient {
+    if (self.kaaClient) {
+        [self.kaaClient start];
+    } else {
+        self.kaaClient = [KaaClientFactory clientWithContext:[[DefaultKaaPlatformContext alloc] init] stateDelegate:self];
+        [self.kaaClient start];
+    }
+    self.verifiersTokens = [self.kaaClient getConfiguration];
+}
+
+- (void)attachUser:(User *)user delegate:(id<UserAttachDelegate>)delegate {
+    NSLog(@"Attaching user...");
+    [self.kaaClient attachUserWithVerifierToken:[self getKaaVerifiersTokenForUser:user] userId:user.userId accessToken:user.token delegate:delegate];
```

Comment:
Code style: Line is too long. 
It will looks better to move each method parameter to the new line.